app/reports/reports_list.tsx

```diff
@@ -1,35 +1,58 @@
-// A server page containing client component and nested server component
 import { api } from "@/services/url";
+import Link from "next/link";
+import { Report } from "../types";
+import axios from "axios";
+import { useState, useEffect } from "react";
+import Loading from "../loading";
 
 export async function getData() {
-    const apiKey = process.env.AF_REPORTS_DB_API_KEY;
+    const apiKey = process.env.NEXT_PUBLIC_AF_REPORTS_DB_API_KEY;
     // Temporary till we implement tokens in portal
     const studentId = process.env.NEXT_PUBLIC_STUDENT_ID;
```

Comment:
shouldn't this be obtained from `user_id` in { loggedIn, user_id } = useAuth()

Reply:
Yeah it will be changed once the testing from the tech team is done. I haven't changed it because it might become complex for some of us to have a valid student_id with reports.